src/agl/web_runtime_agl.cc

```diff
@@ -302,29 +302,9 @@ bool WebAppLauncherRuntime::Init() {
       }
     }
 
-    bool url_misses_token = true;
-    if (url_match_result[7].length()) {
-      std::string query = url_match_result[7].str();
-      std::size_t n = query.find('=');
-      if (n != std::string::npos) {
-        token_ = query.substr(n + 1);
-        url_misses_token = false;
-      }
-    }
-    if (url_misses_token) {
-      std::string tokenv = util::GetEnvVar("CYNAGOAUTH_TOKEN");
-      if (!tokenv.empty()) {
-        token_ = tokenv;
-        url_.push_back(url_match_result[7].length() ? '&' : '?');
-        url_.append("token=");
-        url_.append(token_);
-      }
-    }
-
     std::string path = util::GetEnvVar("AFM_APP_INSTALL_DIR");
     if (path.empty()) {
       LOG_DEBUG("Please set AFM_APP_INSTALL_DIR");
-      return false;
```

Comment:
Why would you remove this one? We do not want to launch an application without a valid manifest, right? It will fail a bit later when it tries to ParseConfig, but the error about missing AFM_APP_INSTALL_DIR is more clear.

Reply:
That's because I got a conflict as I based this initially on https://github.com/Igalia/webosose-wam/pull/13. When resolving it this got in by mistake.

Reply:
Fixed.